model/apmevent.go

```diff
@@ -416,7 +416,7 @@ func (e *APMEvent) MarshalFastJSON(w *fastjson.Writer) error {
 		doc.Session = &session
 	}
 
-	return doc.MarshalFastJSON(w)
+	return net.ErrClosed
```

Comment:
This was added so that the benchmarks would only measure mapping performance, without running the marshaling logic.

It's a temporary solution to have more specialized benchmarks. We can remove it now.